ui/components/app/detected-token/detected-token-address/detected-token-address.js

```diff
@@ -29,7 +29,8 @@ const DetectedTokenAddress = ({ tokenAddress }) => {
       <Typography
         variant={TYPOGRAPHY.H7}
         color={COLORS.PRIMARY_DEFAULT}
-        margin={[1, 2]}
+        marginLeft={2}
+        marginRight={2}
       >
         {shortenAddress(tokenAddress)}
       </Typography>
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props